applications/asset_tracker_v2/src/events/app_module_event.c

```diff
@@ -110,7 +110,10 @@ COMMON_EVENT_INFO_DEFINE(app_module_event,
 
 #endif /* CONFIG_PROFILER */
 
+const uint8_t app_module_event_flags = (IS_ENABLED(CONFIG_APP_EVENTS_LOG))
```

Comment:
static is needed

Reply:
if we want to keep the variable - check in map if it is optimized out (with limited scope it should be).
I also recommend using short names (c is a spartan language ;)). Try to keep declaration in one line.
e.g.
```
const uint8_t flags = IS_ENABLED(CONFIG_APP_EVENTS_LOG) ? BIT(EVENT_FLAGS_INIT_LOG_ENABLE) : 0;
```

Reply:
Yes it is optimized :)